ur_robot_driver/doc/usage/startup.rst

```diff
@@ -72,6 +73,23 @@ Depending on the :ref:`robot control mode<operation_modes>` do the following:
 
 .. _continuation_after_interruptions:
 
+Verify calibration info is being used correctly
```

Comment:
When using a wrong calibration, the driver will output an error. I always have stdout coloring enabled for ROS, so I usually look for a red error message during startup to verify that I have specified the correct calibration. Maybe we should mention that?

Reply:
Sure. added in [8437787](https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver/pull/1214/commits/84377872491216a4220a50cc602610ce4de7c746).